pkg/liquidity-mining/contracts/interfaces/ILiquidityGauge.sol

```diff
@@ -14,24 +14,16 @@
 
 pragma solidity ^0.7.0;
 
-import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/IERC20.sol";
-
 // For compatibility, we're keeping the same function names as in the original Curve code, including the mixed-case
 // naming convention.
 // solhint-disable func-name-mixedcase
 
-interface ILiquidityGauge is IERC20 {
-    function initialize(address lpToken) external;
-
-    function lp_token() external view returns (IERC20);
-
-    function deposit(uint256 value, address recipient) external;
-
-    function withdraw(uint256 value) external;
```

Comment:
Why are we moving these out of the base interface?

Reply:
This is because the `StakelessGauge` implements the `ILiquidityGauge` interface and doesn't contain these concepts. I'm trying to keep this interface as just having functions which are shared across _all_ mainnet gauges.